tensorflow/core/kernels/mkl_tfconv_op.h

```diff
@@ -110,32 +114,16 @@ class MklToTfOp : public OpKernel {
                                   input_number, output_shape, &output_tensor));
       CHECK_NOTNULL(output_tensor);
 
-      // Do we need to reorder MKL layout into TensorFlow layout?
+      // Check if input needs to be reordered
       if (input.IsReorderNeeded(OUTPUT_TF_MD)) {
-#ifdef ENABLE_MKLDNN_V1
-        std::vector<primitive> net;
-        std::vector<MemoryArgsMap> net_args;
-        // Insert reorder between MKL layout and TensorFlow layout.
-        // TODO(bhavanis): Refactor CheckReorderToOpMem() to directly insert
-        // the reorder primitive
-        bool status = input.CheckReorderToOpMem(output_tf_md, output_tensor,
-                                                net, net_args, cpu_engine);
-        OP_REQUIRES(
-            context, status,
-            errors::Internal("MklToTfOp: Failed to create input reorder"));
-
-        ExecutePrimitive(net, &net_args, cpu_engine);
```

Comment:
`ExecutePrimitive` is not called anymore. Is this intentional?

Reply:
Actually, this version of `CheckReorderToOpMem()` both creates and executes the reorder primitive: i.e. https://github.com/tensorflow/tensorflow/blob/063a8303f69645e1da610ef8a5bbfcfc782e42bb/tensorflow/core/util/mkl_util.h#L1680 calls https://github.com/tensorflow/tensorflow/blob/063a8303f69645e1da610ef8a5bbfcfc782e42bb/tensorflow/core/util/mkl_util.h#L1613 which in turn executes reorder.

Reply:
Got it. Thank you for the explanation! :)